### DISCUSSION TITLE
Can I prevent automated config creation?

### DISCUSSION THREAD

**Discussion by charjr** [phpactor/phpactor#2643] [Q&A] [answered]
I have a config file set up in one of the standard locations:
`/home/user/.config/phpactor/phpactor.json`

Every time I work on a project, if a .phpactor.json does not exist in the root of the project, it creates one.

Is there any way to avoid this automated process? If I need specific overrides per project I would prefer to do so manually.

**Comment by dantleech** [comment, marked as answer]
I would add a global gitignore for that file, but happy to accept PRs for disabling the autocreation

**Reply by charjr** [reply]
Thank you for the prompt answer, I did not know about global ignores. That is quite handy.

If I find the time I may look into the auto-creation, thank you.
